swim/member.go

```diff
@@ -123,11 +129,27 @@ type Change struct {
 	Address           string `json:"address"`
 	Incarnation       int64  `json:"incarnationNumber"`
 	Status            string `json:"status"`
+	Tombstone         bool   `json:"tombstone,omitempty"`
```

Comment:
Do we have a test for the omitempty case? (Unmarshal to map[string]interface{} and checking for the field would suffice)